web/src/components/merchant/sign-up/SignUpCard.tsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import FormRow from 'components/common/FormRow';
+import GroupBuyMerchantHeader from 'components/common/GroupBuyMerchantHeader';
+import Form from 'muicss/lib/react/form';
+import Row from 'muicss/lib/react/row';
+import styled from 'styled-components';
+
+const CardContainer: React.FC = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  background: var(--pale-gray);
+  border-radius: 30px;
+  max-width: 670px;
+  padding: 40px;
+  box-shadow: 4px 4px 10px rgba(0, 0, 0, 0.25);
+`;
+
+const StyledRow = styled(Row)`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+
+  margin: 15px 0;
+`;
+
+const SubmitButton = styled.input`
```

Comment:
Shouldnt this be a `button` instead? But consider using this [Button](https://www.muicss.com/docs/v1/react/buttons) and modifying the color and border-radius instead! This way we don't need the `cursor: pointer` and `outline: none` styles too

Reply:
I thought using input type=submit is easier to submit the form data.

Reply:
But okay I think Button better can put onaction also.

Reply:
Haha technically button in a form also also do `type="submit"` but it will trigger a page refresh which is not desired! So yep just handle form submissions with the `onClick` :)